app/backend/controllers/auth.js

```diff
@@ -0,0 +1,9 @@
+module.exports.authenticationBinder = () => {
```

Comment:
We might change the name of the function to something like `isAuthenticated`, and instead of returning a function, the function itself may do the necessary checks, instead of returning a function doing the authentication control.